webapi/src/main/scala/org/knora/webapi/routing/admin/lists/DeleteListItemsRouteADM.scala

```diff
@@ -96,4 +97,32 @@ class DeleteListItemsRouteADM(routeData: KnoraRouteData)
         )
       }
     }
+
+  /**
+   * Deletes all comments from child list node.
+   */
+  private def deleteListChildNodeComments(featureFactoryConfig: FeatureFactoryConfig): Route =
+    path(ListsBasePath / "deletecomments" / Segment) { iri =>
```

Comment:
see my comment in the `lists.md` about the name of the route.

Reply:
Sorry, but which one do you mean?

Reply:
Now I don't see it anymore (maybe I wanted to add the comment and forgot...). I thought it was somehow duplicated information to use `DELETE` with `.../deletecomments/...`. So, I wanted to suggest to use:
`DELETE .../v2/lists/comment/{ListNodeIRI}`
In that case the URL pattern would be consistent with deleting comments from properties or classes. But maybe there is another pattern in the lists route that you are implementing already.

Reply:
Now I don't see it anymore (maybe I wanted to add the comment and forgot...). I thought it was somehow duplicated information to use `DELETE` with `.../deletecomments/...`. So, I wanted to suggest to use:
`DELETE .../v2/lists/comment/{ListNodeIRI}`
In that case the URL pattern would be consistent with deleting comments from properties or classes. But maybe there is another pattern in the lists route that you are implementing already.